.github/workflows/coverity-reusable.yml

```
@@ -0,0 +1,51 @@
+        required: true
+        type: string
+      extra-deps:
+        required: false
+        type: string
+        default: ""
+jobs:
+  coverity:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 🔄 Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.9)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)